### PR TITLE
Update stackdriver dependency to released version.

### PR DESCRIPTION
**Pull request by yegle**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	cloud.google.com/go/storage v1.12.0
 	contrib.go.opencensus.io/exporter/ocagent v0.7.0
 	contrib.go.opencensus.io/exporter/prometheus v0.2.0
-	contrib.go.opencensus.io/exporter/stackdriver v0.13.4
+	contrib.go.opencensus.io/exporter/stackdriver v0.13.5
```
Comment (whaught) [inline go.mod:10]: Remove line 85 of this file?

**Comment by ryanwolcott321** [comment]
Approve

File: go.mod (hunk 1 above)
Comment (yegle) [reply go.mod:10]: Done.

**Review by mikehelmick — APPROVED** [review]

**Pull request merged**